src/components/multiform/MultiformPage.vue

```diff
@@ -0,0 +1,232 @@
+    mutationForm(multiForm, formData, stepNum) {
+      let fileValue = formData.filter((item) => item.name === "img")[0].value;
+      multiForm.forEach((item) => {
+        if (item.step === stepNum) {
+          item.data = JSON.parse(JSON.stringify(formData));
+          item.data.map((child) => {
+            if (child.name === "img") {
+              child.value = [...fileValue];
+            }
+          });
+        }
+      });
+    },
+    onAddCompany() {
+      this.formData.push(JSON.parse(JSON.stringify(formSecondStep)));
+    },
+    onRemoveCompany(index) {
+      this.formData.splice(index, 1);
+    },
+    onInputImg(data) {
+      this.formData.forEach((i) => {
+        if (i.name === "img") {
+          i.value = data;
+        }
+      });
+    },
+    onRemoveImage(index) {
+      this.formData.forEach((i) => {
+        if (i.name === "img") {
+          i.value.splice(index, 1);
+        }
+      });
+    },
+    onInput(value, index) {
+      if (this.formData[index].name === "salary") {
+        this.formData[index].value = +value;
+      } else {
+        this.formData[index].value = value;
+      }
+      this.formData[index].msg = "";
+    },
+    onChangeCompanyForm(value, indexChild, index) {
+      this.formData[index].fields[indexChild].value = value;
+      this.formData[index].fields[indexChild].msg = "";
+    },
+    onSelectJob(option) {
+      this.formData.forEach((i) => {
+        if (i.name === "jobs") {
+          let value = i.value.filter((c) => c.name === option.name);
+          if (value.length <= 0) {
+            i.value.push(option);
+            i.list.map((opt) => {
+              if (opt.name === option.name) {
+                opt.isSelected = true;
+              }
+            });
+          }
+        }
+      });
+    },
+    onRemoveJob(option) {
+      this.formData.forEach((i) => {
+        if (i.name === "jobs") {
+          i.value.splice(i.value.indexOf(option), 1);
+          i.list.map((opt) => {
+            if (opt.name === option.name) {
+              opt.isSelected = false;
+            }
+          });
+        }
+      });
+    },
+    toFormJSON() {
+      let form = {};
+      this.firstForm.forEach((item) => {
+        let data = {};
+        if (item.name != "img" && item.name != "jobs") {
+          form[item.name] = item.value;
+        } else {
+          for (let i = 0; i < item.value.length; i++) {
+            data[i] = item.value[i];
+          }
+          form[item.name] = data;
+        }
+      });
+
+      form["companies"] = {};
+      let data = [];
+      let value = {};
+
+      this.secondForm.forEach((item) => {
+        item.fields.forEach((c) => {
+          value[c.name] = c.value;
+        });
+        data.push(value);
+      });
+
+      for (let i = 0; i < data.length; i++) {
+        form["companies"][i] = data[i];
+      }
+
+      this.thirdForm.forEach((item) => {
+        form[item.name] = item.value;
+      });
+
+      return form;
```

Comment:
cái hàm formToJSON này thấy đang không hợp lí lắm nhé, giả sử sau này thêm 2-3 form nữa vào list thì tự dưng mình phải quay lại cái hàm này để for tiếp các form mới thêm ra để convert ư?